index.html

```diff
@@ -129,45 +145,100 @@ <h2 class="card-title">Tonic</h2>
+          </li>
+          <li class="github">
+            <a href="#"><img src="./logos/github.png" alt="github-logo" /></a>
+          </li>
+          <li class="angelistic">
+            <a href="#"><img src="./logos/angelist.png" alt="github-logo" /></a>
+          </li>
+        </ul>
+        <div class="button1"><a href="#">Get My Resume</a></div>
 
-            <div class="skill-title">
-              <div class="container1">
-                <p class="skill">Languages</p>
-                <p class="dropdown">--</p>
-              </div>
-             <div class="container1">
-              <div class="javascript"><img src="./logos/javascript2.png" alt="github-logo"/>Javascript</div>
-              <div class="html"><img src="./logos/html.png" alt="github-logo"/>Html</div>
-              <div class="css"><img src="./logos/css.png" alt="github-logo"/>Css</div>
-              <h2 class="skill2">Frameworks</h2>
+        <div class="skill-title">
+          <div class="container1">
+            <p class="skill">Languages</p>
+            <p class="dropdown">--</p>
+          </div>
+          <div class="container1">
+            <div class="javascript">
+              <img src="./logos/javascript2.png" alt="github-logo" />Javascript
+            </div>
+            <div class="html">
+              <img src="./logos/html.png" alt="github-logo" />Html
+            </div>
+            <div class="css">
+              <img src="./logos/css.png" alt="github-logo" />Css
+            </div>
+            <h2 class="skill2">Frameworks</h2>
 
-              <h2 class="skill2">Skills</h2>
-             </div> 
-              
+            <h2 class="skill2">Skills</h2>
           </div>
+        </div>
+      </section>
+
+      <section class="contactSection" id="contactSection">
+        <h1 class="title4">Contact Me</h1>
+        <p class="introText">
+          If you have an application you are interested in developing, 
+          a feature that you need built or a project that needs coding.
+          I’d love to help with it 
+            </p>
+          <div class="contactForm">
+            <form
+              action="https://formspree.io/f/xjvlpydr"
+              class="form"
+              method="post"
+            >
+              <input
+                type="text"
+                name="user_name"
+                id="user_name"
+                placeholder="Full name"
+                maxlength = "20"
+                
+              />
+              <input
+                type="email"
+                name="user_email"
+                id="user_email"
+                placeholder="Email address"
+                required
+              />
+              <textarea
+                name="message"
+                id="message"
+                cols="30"
+                rows="10"
+                placeholder="Enter Text Here"
+                maxlength="30"
+              ></textarea>
```

Comment:
- [ ] Kindly be noticed that the max-length for the textarea should be 500 and it should have a required attribute.